packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-reset-password/amplify-reset-password.component.html

```diff
@@ -6,7 +6,9 @@
       data-amplify-fieldset
       [disabled]="authenticator.isPending"
     >
-      <h3 class="amplify-heading">{{ this.headerText }}</h3>
+      <amplify-slot name="reset-password-header">
+        <h3 class="amplify-heading">{{ this.headerText }}</h3>
+      </amplify-slot>
```

Comment:
Looks good with the default text for now until we decide to get rid of the default text.